lib/css-assembler.js

```diff
@@ -2,6 +2,7 @@ var _ = require('lodash'),
     Async = require('async'),
     Fs = require('fs'),
     Path = require('path'),
+    upath = require('upath'),
```

Comment:
It is already there - https://github.com/bigcommerce/stencil-cli/blob/master/package.json#L65